packages/create-atomic/src/plopfile.ts

```diff
@@ -77,21 +77,21 @@ export default function (plop: NodePlopAPI) {
         ],
       },
       function installPackagesPrompt() {
-        return 'Installing packages 🚀';
+        return '🚀  Installing packages...';
```

Comment:
You should check the terminal supports an emoji before using it.
Check https://github.com/koala-interactive/is-emoji-supported (no transitive deps attached so it's pretty light)

Reply:
I mean it could just remove the emojis if they can cause an issue. The gain is pretty low

Reply:
> I mean we could just remove the emojis if they can cause an issue. The gain is pretty low

As you prefer really, the choice is yours.